content/SCALE/SCALEUIReference/Apps/LaunchDockerImageScreens.md

```diff
@@ -8,6 +8,9 @@ tags:
  - scaleapps
 ---
 
+{{< toc >}}
+
+
```

Comment:
Line 18:
"The docker image wizard includes 12 configuration screens and a **[Confirm Options]()** screen that displays a summary of some of the setting options configured."

The Confirm Options link connect to this same page.
Is it supposed to do that?